Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -0,0 +1,110 @@
+:root {
+    /*colors*/
+    --primary: #0C6594;
+    --highlight: #DEF3FE;
+    /*not needed?*/
+    --gray: #C4C4C4;
+    /*not needed?*/
+    --light-gray: #E1E1E1;
+    --menu-background: #F9F9F9;
+    --hover: #218FC9;
+    --border: #979797;
+    --yellow: #DFAF00;
+    --red: #DA2222;
+    --green: #4AD439;
+    --focus: #23ADF8;
+    --header: #F2F2F2;
+
+    --dropdown-scrollbar-track: #E3E3E3;
+    --scrollbar-track: #D3D3D3D3;
+    --background: #FFFFFF;
+
+    /*remotion-themed*/
+    --remotion-themed-border: thin solid var(--border);
+    --remotion-themed-border-focus: thin solid var(--focus);
+    --remotion-themed-border-radius: 0.125em;
```

Comment:
border width and border radius should be the same unit and probably both px. We should not rely on "thin" for border.